.NET/Microsoft.Recognizers.Definitions/French/DateTimeDefinitions.cs

```diff
@@ -193,6 +193,7 @@ public static class DateTimeDefinitions
 		public const string GeneralEndingRegex = @"^[.]";
 		public const string MiddlePauseRegex = @"^[.]";
 		public const string DurationConnectorRegex = @"^[.]";
+		public const string OrRegex = @"^[.]";
```

Comment:
Add TODOs for the other languages.

Reply:
the TODOs are all in the yaml file. The definitions class is auto-generated.